.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Treat all text files without a CR (as we have Docker)
+* text eol=lf
```

Comment:
Why don't you just make it a default setting on your machine?

Reply:
If other Windows users want to build the demo, they will be broken without this setting.  Making it global on my machine is too broad a change to support this demo.